web/views/statuses.erb

```diff
@@ -26,6 +32,30 @@
 </style>
 
 <h3 class="wi">Recent job statuses</h3>
+<div class="max-items col-sm-3 col-sm-offset-9">
```

Comment:
To avoid hard-coding and to improve configurability, could I ask you to rewrite using an iterator over the array of possible values? (In the first comment at top)